.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
+  workflow_dispatch:
+
+
+jobs:
+  build_package_deploy:
+    name: Deploy Module
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PowerShell module cache
+        id: cacher
+        uses: actions/cache@v3
+        with:
+          path: "~/.local/share/powershell/Modules"
+          key: ${{ runner.os }}-PSModules
+      - name: Setup
+        shell: pwsh
+        run: |
+          ./Tools/setup.ps1
+          Invoke-Build -Task ShowInfo
+      - name: Build
+        shell: pwsh
+        run: |
+          Invoke-Build -Task Clean, Build
+      - name: Package
+        shell: pwsh
+        run: |
+          Invoke-Build -Task Package
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v3
```

Comment:
Can we add ` if-no-files-found: error` (default is warning")?
https://github.com/actions/upload-artifact#customization-if-no-files-are-found

Reply:
The package step is 'new', it's so we can generate the zip file for the module. I'm not sure yet if it's even worth keeping once we add the release mechanism to GitHub. But will make the changes. Good for it to fail if not found.